src/components/video/video.jsx

```diff
@@ -35,44 +40,53 @@ class Video extends React.Component {
             onReady: video => {
                 video.bind('play', () => {
                     this.setState({videoStarted: true});
-                    this.props.onVideoStart();
+                    if (this.props.onVideoStart) {
```

Comment:
I am wondering whether it would be better to give `onVideoStart` a default value. But I don't mind it either way